lib/Parser/Parse.cpp

```diff
@@ -2417,6 +2415,23 @@ IdentPtrList* Parser::GetRequestedModulesList()
     return m_currentNodeProg->AsParseNodeModule()->requestedModules;
 }
 
+void Parser::VerifyModuleLocalExportEntries()
+{
+    ModuleImportOrExportEntryList* localExportRecordList = GetModuleLocalExportEntryList();
+    if (localExportRecordList != nullptr)
+    {
+        localExportRecordList->Map([=](ModuleImportOrExportEntry exportEntry) {
```

Comment:
Is it possible for the pidRefStack field to be null here?

Reply:
Yes:
1. offsetForError defaults to 0 if not provided and in that case pidRefStack is left as a nullptr

2. I only provide offsetForError when the export statement is not a function/class/variable declaration as when it is a declaration you can't hit this error so seemed to be unnecessary work to handle this in those cases.

Reply:
I see. Thanks.